src/DefaultBuilder/src/WebApplicationBuilder.cs

```diff
@@ -71,7 +70,7 @@ internal WebApplicationBuilder(Assembly? callingAssembly, string[]? args = null)
         /// <summary>
         /// A collection of services for the application to compose. This is useful for adding user provided or framework provided services.
         /// </summary>
-        public IServiceCollection Services { get; } = new ServiceCollection();
+        public IServiceCollection Services { get; } = default!;
```

Comment:
```suggestion
        public IServiceCollection Services { get; }
```

You can do this since it's set in the ctor, right?

Reply:
It was complaining in the IDE but maybe it was just laggy.